### PR TITLE
Fixes #9862: Generation hooks are not packaged

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9862

### CODE REVIEW

**Comment by amousset** [comment]
I don't really like the `/opt/rudder/hooks.d` path.

File: rudder-webapp/debian/dirs [hunk 1]
```diff
@@ -9,6 +9,7 @@ opt/rudder/etc/plugins
 opt/rudder/share/tools
 opt/rudder/share/plugins
 opt/rudder/share/upgrade-tools
+opt/rudder/hooks.d
```
Comment (VinceMacBuche) [inline rudder-webapp/debian/dirs]: opt/rudder/etc/hooks.d

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -207,6 +207,7 @@ mkdir -p %{buildroot}%{rudderdir}/share/plugins/
 mkdir -p %{buildroot}%{rudderdir}/share/upgrade-tools/
 mkdir -p %{buildroot}%{rudderdir}/share/certificates/
 mkdir -p %{buildroot}%{rudderdir}/share/selinux/
+mkdir -p %{buildroot}%{rudderdir}/hooks.d/
```
Comment (VinceMacBuche) [inline rudder-webapp/SPECS/rudder-webapp.spec]: /etc/hooks.d

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 3]
```diff
@@ -249,6 +250,8 @@ cp %{_sourcedir}/rudder-sources/rudder/rudder-web/src/main/resources/rudder-vhos
 cp %{_sourcedir}/rudder-sources/rudder/rudder-web/src/main/resources/rudder-vhost-ssl.conf %{buildroot}/etc/%{apache_vhost_dir}/rudder-vhost-ssl.conf
 cp %{_sourcedir}/rudder-sources/rudder/rudder-web/src/main/resources/apache2-sysconfig %{buildroot}/etc/sysconfig/rudder-apache
 
+cp -r %{_builddir}/rudder-sources/rudder/rudder-core/src/main/resources/hooks.d %{buildroot}%{rudderdir}/
```
Comment (VinceMacBuche) [inline rudder-webapp/SPECS/rudder-webapp.spec]: /etc at the end

File: rudder-webapp/debian/rules [hunk 4]
```diff
@@ -75,6 +75,7 @@ binary-arch: install
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-sources/rudder/rudder-core/src/main/resources/ reportsInfo.xml /opt/rudder/etc/
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-sources/rudder/rudder-web/src/main/resources/ load-page/  /opt/rudder/share/
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-sources/rudder/rudder-web/src/main/resources/ rudder-apache-common.conf /opt/rudder/etc/
+	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-sources/rudder/rudder-core/src/main/resources/ hooks.d/  /opt/rudder/
```
Comment (VinceMacBuche) [inline rudder-webapp/debian/rules]: /opt/rudder/etc

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Pull request merged**